web/public/js/services/intercom-service.js

```diff
@@ -8,6 +8,11 @@ angular.module('cpZenPlatform').factory('intercomService', function ($localStora
     return _.contains(user.roles, 'cdf-admin');
   }
 
+  function userIsNinja(user) {
+    var initUserTypeStr = JSON.parse(user.initUserType);
```

Comment:
this is initialized outside function scope too. I don't think you need to do it twice